components/studies/ui/src/main/resources/PhenoTips/StudyClass.xml

```diff
@@ -125,6 +125,46 @@
   <object>
     <name>PhenoTips.StudyClass</name>
     <number>0</number>
+    <className>PhenoTips.StudyVisibilityClass</className>
```

Comment:
I don't think you need to add this object to the `Study`__`Class`__, since this isn't an actual Study. Perhaps you want to add it to the `StudyTemplate`?